CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ Mapbox welcomes participation and contributions from everyone.
 # 10.4.0-beta.1
 
 ## Features ✨ and improvements 🏁
-* Refactor scheduling logic for render thread in general slightly improving rendering performance. ([#1068](https://github.com/mapbox/mapbox-maps-android/pull/1068))
+* Refactor scheduling logic for render thread slightly improving rendering performance. ([#1068](https://github.com/mapbox/mapbox-maps-android/pull/1068))
```

Comment:
Do we need to mention `slightly`, looks quite vague. 
could it be `Refactor scheduling logic for render thread to improve rendering performance`.

Reply:
agree here